packages/addons/service/vdr-addon/package.mk

```diff
@@ -18,11 +18,11 @@
 ################################################################################
 
 PKG_NAME="vdr-addon"
-PKG_VERSION="8.0"
+PKG_VERSION="2.2"
```

Comment:
this version is more or less just for the build dir that it isn't `vdr-addon-` for nothing else it is used